src/react-app-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/* eslint-ignore */
+// / <reference types="react-scripts" />
```

Comment:
I'm actually not sure what is this file and whether it is required

Reply:
was this created by create-react-app? I don't think it's needed.

Reply:
Yes, it was, ok will try to remove this

Reply:
If it just keeps regenerating it, then we can keep it, it doesn't hurt.